IDEAS.org

```diff
@@ -31,14 +39,38 @@ Signed git tags should be highlighted.
   monitor key servers to make sure keys are up to date and modules signed with
   revoked keys have proper notifications for users
 
+***** Proposal(s):
+
++ To be discussed with some of their people of infrastructure and monitoring 
+  team. It involves setting up monitoring entries to those keyserver URL's and 
+  use this information to match the usage of this key it to the list of users
+  that downloaded/use the module. This can be achieved by expanding current
+  capabilities of the puppetlabs.com user accounts. Only registered users could
+  really benefit from this though. It also gives a better "popcon" statistics 
+  usage. 
++ It might be also implemented in the puppet server itself. like any linux 
+  distro's way of gathering software usage statistics (like debian's "popcon")
+  the puppet server could have a feature where modules downloaded directly from
+   the forge, send some anonymous statistics about the module usage.  
```

Comment:
Since what we're talking about is a central hub for Puppet modules, a `popcon` shouldn't be necessary -- the Forge could just collect metrics. popcon shouldn't be necessary unless you want to mirror the contents of Forge 2.0 (which might be a good idea?), as you'd then end up with multiple independent mirrors each collecting their own metrics.